schemas/com.vimeo/cuepoint/jsonschema/1-0-0

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Vimeo Cue Point.",
+  "self": {
+    "vendor": "com.vimeo",
+    "name": "cuepoint",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "format": "uuid",
+      "description": "The ID of the cue point."
+    },
+    "time": {
+      "type": "integer",
+      "description": "The location of the cue point in seconds.",
+      "minimum": 0,
+      "maximum": 86400
```

Comment:
Should the maximum be 2147483647 to reflect the maximum for other playback time properties? (I also see that the `startTime` property in the `chapter` schema has a low maximum but the description there says that it is max 24 hours).

Reply:
I think that might be sensible to do so for `chapter` too. I got the 24 hours from [here](https://developer.vimeo.com/api/upload/videos), but I'm not 100% sure now if that's the max possible, or just for uploading via the API.